src/components/ExperimentalFilters/ExperimentalFilters.tsx

```diff
@@ -41,32 +41,36 @@ type Right =
 
 type RightInput = {
   value: string;
-  conditionValue: { label: string; value: string; type: "text" | "number" };
+  conditionValue: {
+    label: string;
+    value: string;
+    type: "text" | "number";
+  } | null;
 };
 
 type RightMultiselect = {
   value: { label: string; value: string }[];
-  conditionValue: { label: string; value: string; type: "multiselect" };
+  conditionValue: { label: string; value: string; type: "multiselect" } | null;
```

Comment:
**suggestion**: don't you want to create separate type for it? you repeat this one many times? you can create generic here

Reply:
I want to first make it work with the dashboard - those types should be refactored before production use 🙂